workers/unity/Assets/Gdk/Core/Systems/MetricSendSystem.cs

```diff
@@ -0,0 +1,73 @@
+using System.Collections.Generic;
+using Unity.Entities;
+using UnityEngine;
+using Improbable.Gdk.Core.Config;
+
+namespace Improbable.Gdk.Core
+{
+    public class MetricSendSystem : ComponentSystem
+    {
+        private WorkerBase worker;
+
+        private float timeElapsedSinceUpdate = 0.0f;
+
+        private List<float> fpsMeasurements = new List<float>();
+        
+        protected override void OnCreateManager(int capacity)
+        {
+            base.OnCreateManager(capacity);
+            worker = WorkerRegistry.GetWorkerForWorld(World);
+        }
+
+        private float DefaultLoadCalculation(float fps)
+        {
+            float targetFps = Application.targetFrameRate;
+            return Mathf.Max(0.0f, (targetFps - fps) / (targetFps - 20.0f));
```

Comment:
The `20.0f` should be some multiple of `targetFps`. I.e - we say load = 1 when you are at 50% of your target FPS or similar.